packages/web/components/Seeds/modals.tsx

```diff
@@ -687,95 +667,68 @@ export const JoinTheFund = () => (
-            announcement post
-          </Link>
-          , pitch video or dig through the website itself.
-        </Text>
-      )}
-      <Box onClick={() => setIsOpenInfinity(!isOpenInfinity)} my={4}>
-        <details>
-          <summary>Will you just keep on minting Seeds to infinity?</summary>
-        </details>
-      </Box>
-      {isOpenInfinity && (
-        <Text>
-          Not likely! The idea is that MetaGame will go
-          <Link
-            ml={1}
-            isExternal
-            color="gray.500"
-            textDecoration="underline"
-            mr={1}
-            href="https://wiki.metagame.wtf/docs/how-does-it-work/phases-of-metagame"
-          >
-            through 3 phases
-          </Link>
-          ; starting with Seeds, growing into Trees in
-          <em>Phase II</em> aka <em>The Growth Phase </em>before transitioning
-          to Phase III - where Trees bear the fruits of our labor &amp; we’re
-          able to reward contributors with MetaGame’s incoming streams without
-          having to mint new tokens.
-        </Text>
-      )}
-      <Box onClick={() => setIsOpenGrow(!isOpenGrow)} my={4}>
-        <details>
-          <summary>Why should the price of Seeds grow?</summary>
-        </details>
-      </Box>
-      {isOpenGrow && (
-        <>
-          <Text>🌊 Seed Watering Streams</Text>
-          <UnorderedList mt={2}>
-            <ListItem>
-              all fees &amp; payments received by MetaGame go into sustaining
-              the price.
-            </ListItem>
-            <ListItem>
-              people subscribe to buying Seeds for any of the above reasons or
-              others.
-            </ListItem>
-            <ListItem>
-              projects we helped incubate share a % of their tokens?
-            </ListItem>
-          </UnorderedList>
-        </>
-      )}
+export const FAQ = () => (
+  <Flex direction="column" w="100%">
+    <CollapsableText title="What have you accomplished so far?">
+      <Text>
+        Many things! Go read the fundraise
+        <Link
+          ml={1}
+          isExternal
+          color="gray.500"
+          textDecoration="underline"
+          mr={1}
+          href="https://metagame.substack.com/p/calling-for-rain"
+        >
+          announcement post
+        </Link>
+        , pitch video or dig through the website itself.
+      </Text>
+    </CollapsableText>
+    <CollapsableText title="Will you just keep on minting Seeds to infinity?">
+      <Text>
+        Not likely! The idea is that MetaGame will go
+        <Link
+          ml={1}
+          isExternal
+          color="gray.500"
+          textDecoration="underline"
+          mr={1}
+          href="https://wiki.metagame.wtf/docs/how-does-it-work/phases-of-metagame"
+        >
+          through 3 phases
+        </Link>
+        ; starting with Seeds, growing into Trees in
+        <em>Phase II</em> aka <em>The Growth Phase </em>before transitioning to
```

Comment:
please add a space to Phase I|
<img width="99" alt="Screenshot 2022-03-06 at 14 24 04" src="https://user-images.githubusercontent.com/35112939/156925191-a8a6361e-4b21-40cc-a1ac-5a34dc05211f.png">